flowzone.yml

```diff
@@ -2084,3 +2154,46 @@ jobs:
           echo "result<<${DELIMITER}" >> $GITHUB_OUTPUT
           echo "${result}" >> $GITHUB_OUTPUT
           echo "${DELIMITER}" >> $GITHUB_OUTPUT
+
+  ###################################################
+  ## configure standard repository settings
+  ###################################################
```

Comment:
Could this logic not be merged with the branch protection steps? Seems like they should fall into the same category and use some of the same code.

Reply:
I thought about that, but decided not to mix them. If you want, we can combine.

Reply:
Can this logic instead use https://github.com/marketplace/actions/repo-settings-sync rather than our own thing?

Reply:
Tough one.. Do we trust `kbrashears5`?

In any case, we would still have to pre-process the inputs, so it would be two steps. I am happy to refactor to use this external code and pin it to a commit if The People decide this is the way..

Reply:
> Tough one.. Do we trust `kbrashears5`?
> 
> In any case, we would still have to pre-process the inputs, so it would be two steps. I am happy to refactor to use this external code and pin it to a commit if The People decide this is the way..

We could always pin to a commit hash to avoid the trust problem, but it looks like we are already supporting fields that the action does not, so I guess we will stick with our method.